README.md

```diff
@@ -26,7 +26,7 @@ For more information, see [opensearch.org](https://opensearch.org/) and the [API
 
 ## User Guide
 
-To get started with the OpenSearch Python Client, see [User Guide](https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md).
+To get started with the OpenSearch Python Client, see [User Guide](https://github.com/opensearch-project/opensearch-py/blob/main/USER_GUIDE.md). This repository also contains [working samples](samples) and [benchmarks](benchmarks).
```

Comment:
@dblock, Please give complete paths for samples and benchmarks like it is for user_guide. To prevent link checker failure.

Reply:
Changed it for this PR, however that feels incorrect because when you're on a branch, you end up navigating to `main` which may have content that has evolved, or even code moved which then will break the link.

Reply:
@dblock, I agree

Reply:
I opened https://github.com/opensearch-project/opensearch-py/issues/538. Note that even with the full link it doesn't pass checking because that URL won't exist until the code is merged.